apps/test-viewer/package.json

```diff
@@ -18,13 +18,13 @@
     "@itwin/core-markup": "^4.0.0",
     "@itwin/core-orbitgt": "^4.0.0",
     "@itwin/core-quantity": "^4.0.0",
-    "@itwin/core-react": "^4.0.0",
+    "@itwin/core-react": "^4.1.0",
     "@itwin/core-telemetry": "^4.0.0",
     "@itwin/ecschema-metadata": "^4.0.0",
     "@itwin/frontend-devtools": "^4.0.0",
     "@itwin/geo-tools-react": "workspace:*",
     "@itwin/imodels-access-frontend": "^3.0.0",
-    "@itwin/imodel-components-react": "^4.0.0",
+    "@itwin/imodel-components-react": "^4.1.0",
```

Comment:
the dep on web-viewer-react can be updated to pull in ^4.0.0, not a dev version

Reply:
Changed `web-viewer-react` to `^4.0.0`